awx/main/access.py

```diff
@@ -1162,6 +1162,52 @@ def can_delete(self, obj):
         #    return True
         return self.can_change(obj, None)
 
+    def get_user_capabilities(self, obj, **kwargs):
+        user_capabilities = super(CredentialAccess, self).get_user_capabilities(obj, **kwargs)
+        user_capabilities['use'] = self.can_use(obj)
```

Comment:
Assuming you have sufficient permissions on the target credential, you need `use` or better permissions on a source credential in order to link it. 

By including `use` in the `user_capabilities` object, we can know if a source credential is linkable.